projects/aca-testing-shared/src/components/viewer/viewer.ts

```diff
@@ -93,4 +94,24 @@ export class Viewer extends Component {
     const count = await this.pdfViewerContentPages.count();
     return count > 0;
   }
+
+  async expectUrlToContain(text: string): Promise<void> {
+    await expect(browser.getCurrentUrl()).toContain(text);
+  }
+
+  async clickDownloadButton(): Promise<void> {
+    const downloadButton: ElementFinder = element(by.id(`app.viewer.download`));
+    await BrowserActions.click(downloadButton);
+  }
+
+  async clickCloseButton(): Promise<void> {
+    const closeButton: ElementFinder = element(by.css('button[data-automation-id="adf-toolbar-back"]'));
+    await BrowserActions.click(closeButton);
+  }
+
+  async viewFile(fileName: string): Promise<void> {
```

Comment:
Seems a bit weird to have a viewFile method inside the Viewer page. Where is this used?